bolivar-coruja-facebook-signup/script.js

```diff
@@ -0,0 +1,94 @@
+// search Elements
+function searchElements() {
+    var date = document.getElementsByClassName("date");
+    var sexClass = document.getElementsByClassName("sex");
+    var gender = document.getElementsByClassName("gender");
+    var input = document.getElementsByClassName("input-sex");
+    var pronoun = document.getElementsByClassName("pronoun");
+    var button = document.getElementsByClassName("button");
+    var email = document.getElementById("email");
+    var senha = document.getElementById("senha");
+    var inputLogin = document.getElementsByClassName("input");
+    return [date, sexClass, gender, input, pronoun, button, email, senha, inputLogin]
+}
+
+function show() {
+    var element = searchElements();
+    if (element[3][2].checked) {
+        element[2][0].style.display = "block";
+    }
+    else {
+        element[2][0].style.display = "none";
+    }
+}
+
+function addEvent() {
+    var element = searchElements();
+    for (i = 0; i < 3; i++) {
+        element[3][i].addEventListener("change", show);
+    }
+    element[5][0].addEventListener("click", enterButton);
+    element[5][1].addEventListener("click", newLoginButton);
+}
+
+function addPronoun() {
+    var element = searchElements();
+    var pronoun = ["Feminino: \"Deseje a ela um feliz aniversário!\"", "Masculino: \"Deseje a ele um feliz aniversário!\"", "Neutro: \"Deseje a ele(a) um feliz aniversário!\""];
```

Comment:
Aqui, poderia pode usar aspas simples `'` e evitar ter que fazer escape das aspas duplas
```suggestion
    var pronoun = ['Feminino: "Deseje a ela um feliz aniversário!"', 'Masculino: "Deseje a ele um feliz aniversário!"', 'Neutro: "Deseje a ele(a) um feliz aniversário!"'];
```